actions/run_action.go

```diff
@@ -86,14 +90,19 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 	if run.Script != "" {
 		run.Script = debos.CleanPathAt(run.Script, context.RecipeDir)
 		if run.Chroot {
-			cmd.AddBindMount(path.Dir(run.Script), "/script")
-			cmdline = []string{fmt.Sprintf("/script/%s", path.Base(run.Script))}
+			scriptpath, err := debos.GetCommandFromString(run.Script)
+			if err != nil {
+				return err
+			}
+			scriptpath = path.Dir(scriptpath)
+			cmd.AddBindMount(scriptpath, "/script")
+			cmdline = []string{strings.Replace(run.Script, scriptpath, "/script", 1)}
 		} else {
 			cmdline = []string{run.Script}
 		}
 		label = path.Base(run.Script)
 	} else {
-		cmdline = []string{"sh", "-c", run.Command}
+		cmdline = []string{run.Command}
```

Comment:
Why this change?

Reply:
we need the same for scripts. Added shell below.